.gitignore

```diff
@@ -0,0 +1,10 @@
+**/*.in
+**/*.o
+**/*.lo
+**/*.la
+**/Makefile
```

Comment:
There're several `Makefile`s in the repo, which is generated using `tmake`. I'd like to remove them and add this line if all existed `Makefile`s can be generated from the source (`make/gen_make/`). Otherwise I'd prefer to remove `Makefile` here.